src/Resources/src/main.ts

```diff
@@ -0,0 +1,48 @@
+import { createApp } from 'vue'
+import { createRouter, createWebHistory } from 'vue-router'
+import { createI18n } from 'vue-i18n' // import from runtime only
+
+import en from '@/locales/en.json';
+import de from '@/locales/de.json';
+
+import '../style/output.css'
+
+import App          from '@/App.vue'
+import MealsMain    from '@/views/MealsMain.vue'
+import Menu         from '@/views/Menu.vue'
+import Dishes       from '@/views/Dishes.vue'
+import Categories   from '@/views/Categories.vue'
+import TimeSlots    from '@/views/TimeSlots.vue'
+import Costs        from '@/views/Costs.vue'
+import Finance      from '@/views/Finance.vue'
+import Balance      from '@/views/Balance.vue'
+import Guest        from '@/views/Guest.vue'
+
+const i18n = createI18n({
+    locale: navigator.language,
+    fallbackLocale: 'en',
+    messages: {
+        en,
+        de
+    }
+})
+
+const router = createRouter({
+    history: createWebHistory(),
+    routes: [
```

Comment:
Do we need to configure all the routes, or can we just configure the ones accessible (based on user role)  to the user?